bin/build-cloud

```diff
@@ -0,0 +1,35 @@
+ #! /usr/bin/env bash
+
+ # DOC: Build civiform/cloud.
```

Comment:
Nit: there's an extra space before these lines at the top of the file.  I think we don't want that